src/modelfree/train.py

```diff
@@ -256,12 +260,23 @@ def maybe_embed_victim(multi_venv, scheduler, env_name, victim_type, victim_path
         assert multi_venv.num_agents == 2
         our_idx = 1 - victim_index
 
+        # If we are actually training an epsilon-ball noise agent on top of a zoo agent
+        if adv_noise_agent_val is not None:
+            base_policy = load_policy(policy_path=victim_path, policy_type=victim_type,
```

Comment:
So the victim is always the base policy? This won't work in asymmetric environments like `KickAndDefend`. I'm inclined to make the base policy configurable.